src/Component/NoteListpage/NoteListpage.css

```diff
@@ -0,0 +1,82 @@
+.search-input-center {
+  width: 30%;
+  height: 4rem;
+  /* background-color: red; */
+  margin: 3rem 0 0 20rem;
+  border: 2px solid var(--light-stone);
+}
+.search-icon {
+  font-size: 2.5rem;
+  margin-left: 1rem;
+  color: var(--light-stone);
+}
+.input-searchbox {
+  border: none;
+  outline: none;
+  width: 70%;
+}
+.notes-with-searchbar-container {
+  flex-basis: 80%;
+  width: 100%;
+  height: 100vh;
+  margin: 8rem 0 0 20rem;
+  /* background-color: aqua; */
+  position: fixed;
+  top: 2.6rem;
+}
+.add-notes-container {
+  width: 58rem;
+  padding: 1rem;
+  gap: 10px;
+  box-shadow: 2px 3px 4px var(--light-stone);
+  margin: 4rem 0 0 20rem;
+  cursor: pointer;
+  /* background-color: white; */
+  /* background-color: aqua; */
+}
+.input-with-pin-icon-container {
+  width: 100%;
+  font-size: 2rem;
+  gap: 10px;
+}
+.title-of-note {
+  width: 100%;
+  border: none;
+  outline: none;
+  padding: 6px;
+  font-size: 1.5rem;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+.note-icon {
+  color: var(--primary-color-dark);
+}
+.details-of-note {
+  width: 100%;
+  border: none;
+  outline: none;
```

Comment:
this seems generic make one class just suggestion for reducing line of code in future also